react/features/app/index.js

```diff
@@ -3,4 +3,9 @@ export * from './actionTypes';
 export * from './components';
 export * from './functions';
 
+// We should import JWT module in order to register reducer and middleware
```

Comment:
"We need to import the jwt module in order to register the reducer and middleware, because the module is not used outside of this feature."